x-pack/plugins/ingest_manager/common/types/models/epm.ts

```diff
@@ -234,6 +239,9 @@ export interface Installation extends SavedObjectAttributes {
   es_index_patterns: Record<string, string>;
   name: string;
   version: string;
+  install_status: EpmPackageInstallStatus;
```

Comment:
IIRC, `status` used to be part of the SavedObject but was removed because everything was always "installed". I think this replaces/restores it. If so, I think it's a `InstalledStatus`

Reply:
I think perhaps the more descriptive `EpmPackageInstallStatus` as we already have `InstallStatus` and `InstallationStatus` for the UI state.